.eslintrc.js

```diff
@@ -1,211 +1,44 @@
 module.exports = {
   // When adding items to this file please check for effects on sub-directories.
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion:  2018,
+    ecmaFeatures: {
+      jsx: true
     },
-    "sourceType": "module"
+    sourceType: "module"
   },
-  "env": {
-    "node": true
+  env: {
+    node: true
   },
-  "plugins": [
+  plugins: [
+    "prettier", // "require("eslint-plugin-prettier")
     "import", // require("eslint-plugin-import")
     "react" // require("eslint-plugin-react")
   ],
-  "extends": [
-    "eslint:recommended",
-    "plugin:mozilla/recommended" // require("eslint-plugin-mozilla")
+  extends: [
+    "plugin:mozilla/recommended", // require("eslint-plugin-mozilla")
+    "plugin:react/recommended",
+    "prettier/react", // require("eslint-plugin-prettier)
+    'prettier/@typescript-eslint',  // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    'plugin:prettier/recommended',  // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always
```

Comment:
https://github.com/prettier/eslint-config-prettier#installation seems to suggest that we want "prettier" here also.  What that actually does, I'm not sure.